light.py

```diff
@@ -140,6 +168,8 @@ async def _on_message_change_level(self, message: ReceiveMessage):
 
     async def _on_message_supported_commands_color_control(self, message: ReceiveMessage):
         self._supported_commands["colorcontrol"] = message.payload
+        _LOGGER.info("UnifyLight: ColorControl %s", self._supported_commands)
+        self._supported_color_modes.add(ColorMode.HS)
```

Comment:
Before adding supported color mode wouldn't it make sense to check if the MoveToHueAndSaturation is available in the supportedcommands. 
We have seen issues for some clusters that the ZPC has a bug, where an empty SupportedCommands is published.

Reply:
Added check